docs/_posts/2020-10-15-apache-hudi-meets-apache-flink.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Apache Hudi meets Apache Flink"
+excerpt: "The design and latest progress of the integration of Apache Hudi and Apache Flink."
+author: wangxianghu
+category: blog
+---
+
+Apache Hudi is a data lake framework developed and open sourced by Uber. Hudi joined the Apache incubator for incubation in January 2019, and was promoted to the top Apache project in May 2020. It is one of the most popular data lake frameworks.
```

Comment:
"originally developed" or "created at Uber" ? Hudi is very much an Apache project.

Reply:
also just denote that Hudi means `Apache Hudi`. Can we say "Apache Hudi (hudi for short) is a data lake ..."

Reply:
> "originally developed" or "created at Uber" ? Hudi is very much an Apache project.

yes better

Reply:
> also just denote that Hudi means `Apache Hudi`. Can we say "Apache Hudi (hudi for short) is a data lake ..."

good idea